src/applications/coronavirus-research/containers/CustomReviewDOBField.jsx

```diff
@@ -3,9 +3,9 @@ import { formatReviewDate } from 'platform/forms-system/src/js/helpers';
 
 export default function CustomReviewDOBField({ children, uiSchema }) {
   return (
-    <div className="review-row">
+    <dl className="review-row">
```

Comment:
Let's change these DLs back to `div[class="review-row"]`. Axe allows for one level of DIVs for nesting, but might throw an error for nested DLs. While experimenting this morning on localhost, I found an extra nesting div was appearing in CustomReviewField.